### ISSUE TITLE
Support for 11G servers

### ISSUE THREAD

**Issue by goetzk** [dell/dellemc-openmanage-ansible-modules#118]
Hi,
I'm wondering if support for 11G servers (C6145's in my case) is feasible or if the BMC is just too limited to make this level of automation workable. Depending on how much work would be required I could try and contribute some code towards the goal.

thanks,
Karl.

**Comment by jagadeeshnv** [comment]
We don't support 11G, neither do we have legacy support from ansible, as this project started recently

**Comment by jagadeeshnv** [comment]
Hi @goetzk 
Sorry for closing, abruptly
Can you give details of the exact use case so that we can look into it

**Comment by goetzk** [comment]
@jagadeeshnv We have a mixed fleet, but its > 50 C6145's plus some newer, already supported, platforms.
Upgrading C6145's causes us the most trouble since we have had to get a particular version of Centos to run the installer.
Some of our systems are so old they require multi step bios upgrades so the process (a problem which caused a couple of bricks before we realised) and can take quite a while. If it was possible to replace all that infrastructure with an Ansible role it would have been very convenient.
